Bottom: 98fa8540e599f17651a25c97403500f4f77a5f63
Top:    30fc01a2d89f2e52c05610db8f94e1b1a67473bc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 07:57:45 +0100

Refresh of use-r-trees-to-find-and-remove

---

diff --git a/src/pour.c b/src/pour.c
index bee0e79..6875ef4 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -823,7 +823,7 @@ subtract_plow (DataTypePtr Data, LayerTypePtr layer, PourTypePtr pour,
 
   if (pg == NULL)
     {
-//      printf ("Hmm, got pg == NULL in subtract_plow\n");
+      printf ("Hmm, got pg == NULL in subtract_plow\n");
       poly_Free (&np);
       return -1;
     }
